hdkeychain/extendedkey_test.go

```diff
@@ -301,11 +301,11 @@ tests:
 		pubKey := extKey.Neuter()
 
 		// Neutering a second time should have no effect.
-		// Test for referencial equality.
+		// Test for referential equality.
 		if pubKey != pubKey.Neuter() {
 			t.Errorf("Neuter of extended public key returned " +
 				"different object address")
-			return
+			continue
```

Comment:
`TestBIP0032Vectors` provided the template for `TestBIP0032Vector4`, but it was cleanest not to force them into one test as vector 4 is specific to the different between "standard" bip32 and our modified variant.
I updated the typo and the unnecessary return here though.